email.sh

```diff
@@ -52,6 +43,13 @@ else
   EVERGREEN=evergreen
 fi
 
+# Fetch previous release tag and get the commits since that tag
+CURRENT_COMMIT_HASH=$(git rev-parse --short HEAD)
+PREVIOUS_TAG=$(git describe --abbrev=0 $CURRENT_COMMIT_HASH\^)
+# get all commits since the previous tag
+git log --no-merges $PREVIOUS_TAG..$CURRENT_COMMIT_HASH --pretty="%h %s" >> body.txt
```

Comment:
nit: I think this should be `>`, not `>>` (on the risk that somehow there's another body.txt, we should overwrite it)